operator/internal/central/extensions/reconcile_tls.go

```diff
@@ -142,12 +152,44 @@ func (r *createCentralTLSExtensionRun) validateAndConsumeCentralTLSData(fileMap
 	if err := r.ca.CheckProperties(); err != nil {
 		return errors.Wrap(err, "loaded service CA certificate is invalid")
 	}
+	if err := r.checkCertificateTimeValidity(r.ca.Certificate()); err != nil {
+		return errors.Wrap(err, "primary CA is not valid at the present time")
+	}
```

Comment:
If we had never checked this here so far then I would feel safer if this was in the feature flag branch for now.